azext_edge/edge/providers/support/shared.py

```diff
@@ -10,7 +10,7 @@
 from ..orchestration.base import ARC_CONFIG_MAP, ARC_NAMESPACE
 from .base import process_events, process_nodes, process_storage_classes
 
-# TODO: This label format doesn't belong here :(
```

Comment:
This TODO wasn't really resolved. The comment is referring to this `shared` module not being appropriate to put label format placeholders.